projects/aca-content/src/lib/components/search/search-save/dialog/edit/saved-search-edit-dialog.component.ts

```diff
@@ -42,15 +42,7 @@ import { UniqueSearchNameValidator } from '../unique-search-name-validator';
   host: { class: 'aca-saved-search-edit-dialog' }
 })
 export class SavedSearchEditDialogComponent {
-  form = new FormGroup({
-    name: new FormControl('', {
-      validators: [Validators.required, forbidOnlySpaces],
-      asyncValidators: [this.uniqueSearchNameValidator.validate.bind(this.uniqueSearchNameValidator)],
-      updateOn: 'blur'
-    }),
-    description: new FormControl('')
-  });
-
+  form: FormGroup;
```

Comment:
could you please set type of that generic FormGroup type? Actually it takes any so it will cause any for values etc. if we will try to read value from that form: 
<img width="1131" alt="Screenshot 2024-11-18 at 12 39 04" src="https://github.com/user-attachments/assets/430230be-08d1-42bc-9e89-49527621a067">

Reply:
Done, I've also applied the same fix for the other dialog for saved searches